packages/css/src/components/Helper.css

```diff
@@ -1,5 +1,8 @@
 .Helper {
+  --text-color: var(--color-granite);
+
   display: block;
+  color: var(--text-color);
 }
```

Comment:
Where is `.Helper` used? I don't see why we'd need a class that adds color and display clock. Wouldn't it be better called `.BaseText` or something?

Reply:
`.Helper` is related to the `Helper` component, which needs some extra properties (like those ones)